<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_finance_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>521</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>id de la caisse         </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="la_caisse"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>somme de la caise  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="la_somme"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>id de l employe      </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="idemp"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>salaire de l employe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="salairedelemp"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>id garage              </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="iddegarage"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>NÂ°facture             </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lafacture"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pb_ajouter">
         <property name="font">
          <font>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>ajouter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>modifier</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>531</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>id de la caisse         </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="idcaisse2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>somme de la caise  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sommecaisse2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>id de l employe      </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="idemp_4"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>salaire de l employe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="salaireemp_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>id garage              </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="idgara_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>facture                 </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fact_2"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="update">
         <property name="font">
          <font>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="tri">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>440</y>
         <width>91</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>trier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="recherche">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>91</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>recherche</string>
       </property>
      </widget>
      <widget class="QPushButton" name="print">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>450</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>imprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supp">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>91</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QTableView" name="tab_caisse">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>741</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="idcaisse2_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>400</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>380</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>IdCaisse</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stat">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>450</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Statistique</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
